Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -16,6 +18,10 @@ void LayoutAnimationsManager::configureAnimation(
     exitingAnimations_[tag] = config;
   } else if (type == "layout") {
     layoutAnimations_[tag] = config;
+  } else if (type == "sharedElementTransition") {
+    sharedTransitionAnimations_[tag] = config;
+    sharedTransitionGroups_.try_emplace(sharedTransitionTag);
+    sharedTransitionGroups_[sharedTransitionTag].push_back(tag);
```

Comment:
minor update would be to use the iterator returned from `emplace` in order to push the value. Sth along these lines should work:
```
map.try_emplace(tag).first->second.push_back(tag)
```
add an appropriate comment that because otherwise it is hardly readable but avoids the second map lookup

Reply:
[unordered_map::operator[]](https://en.cppreference.com/w/cpp/container/unordered_map/operator_at) "is equivalent to `return this->try_emplace(std::move(key)).first->second;`. (since C++17)", so this can be written as simply `sharedTransitionGroups_[sharedTransitionTag].push_back(tag);`, without the `try_emplace`.